### DISCUSSION TITLE
Add a custom "Alerting Engine"

### DISCUSSION THREAD

**Discussion by cyberphor** [Security-Onion-Solutions/securityonion#7960] [Unsupported Versions] [answered]
Hello, 

Is it possible to add a custom "Alerting Engine?"

The SO documentation says there are three (Suricata, Wazuh, and Playbook), but I'd like to add a fourth for my commercial anti-virus products (i.e., McAfee Endpoint Security). 

The specific page of the SO documentation I am referencing is below:
https://docs.securityonion.net/en/2.3/managing-alerts.html#alerting-engines-severity

**Comment by dougburks** [comment, marked as answer]
You might be able to ingest the antivirus logs in Elasticsearch:
https://docs.securityonion.net/en/2.3/syslog.html
https://docs.securityonion.net/en/2.3/filebeat.html#filebeat
https://docs.securityonion.net/en/2.3/elasticsearch.html#parsing

**Reply by cyberphor** [reply]
thanks @dougburks! i plan to setup Winlogbeat on my antivirus manager (McAfee ePO) and create ElastAlert rules via Playbook (this way i'm able to view the antivirus logs in the SOC).
